runtime/queries/koka/highlights.scm

```diff
@@ -0,0 +1,220 @@
+; Function calls
+
+(appexpr
+  function: (appexpr
+    (atom
+      (qidentifier
+        [
+          (qvarid)
+          (qidop)
+          (identifier
+            [(varid) (idop)] @function)
+        ] @function))))
+
+(appexpr
+  function: (appexpr
+    field: (atom
+      (qidentifier
+        [
+          (qvarid)
+          (qidop)
+          (identifier
+            [(varid) (idop)] @function.method)
+        ] @function.method))))
+
+[
+  "initially"
+  "finally"
+] @function.special
+
+; Function definitions
+
+(puredecl
+  (funid
+    (identifier
+      [(varid) (idop)] @function)))
+
+(fundecl
+  (funid
+    (identifier
+      [(varid) (idop)] @function)))
+
+(operation
+  (identifier
+    [(varid) (idop)] @function))
+
+; Identifiers
+
+(puredecl
+  (binder
+    (identifier
+      [(varid) (idop)] @constant)))
```

Comment:
While technically the `val` declarations could be seen as constants, I think the spirit of this scope is more about compile-time constants such as literal strings/numbers/booleans. They tend to have a distinct color in highlight themes, which may look off when used in such a regular quantity for identifiers. Perhaps the `variable` scope suits this case better. 
On the other hand, it would be nice to have `var` declarations stand out from other (immutable) identifers such as `val`s and function parameters. Perhaps we can find a suitable scope for these? The list of scopes at our disposal is not particularly accomodating unfortunately: https://docs.helix-editor.com/themes.html. What about `variable.other`? Or `special`?